docs/codespace_pr_reviews.md

```diff
@@ -0,0 +1,29 @@
+# Using Codesapces to review PRs without pulling a repo to your machine
+
+1. go to the PR
+2. click the `Code` button in the top right corner
+3. click the `Codespaces` tab
+4. `Create new codespace`
+
+   ![Create a new codespace screenshot](https://prod-files-secure.s3.us-west-2.amazonaws.com/bf5cb10c-8e7c-41db-97ca-ded0d1f001c9/49ab6248-8f67-4e79-9ee5-7dee09e03237/Untitled.png)
+
+5. **Wait** for the codespace to finish setting up, you should see stuff happening in the `Terminal`. If you mess with it before it is done it might not work. If you mess this up, yo might see an error about your node version.
```

Comment:
```suggestion
5. **Wait** for the codespace to finish setting up, you should see stuff happening in the `Terminal`. If you mess with it before it is done it might not work. If you mess this up, you might see an error about your node version.
```